src/dispatch/__tests__/DataPlaneClient.test.ts

```diff
@@ -132,4 +132,107 @@ describe('DataPlaneClient tests', () => {
             'd37eb756444ebf6f785233714d6d942f2b20f69292fb09533f6b69556eb0ff2b'
         );
     });
+
+    test('when the endpoint contains a path then the fetch reqest url contains the path prefix', async () => {
+        // Init
+        const endpoint = new URL(
+            `${Utils.AWS_RUM_ENDPOINT.toString()}${'prod'}`
```

Comment:
nit: it looks like the endpoint URL object would look like `<first part of url>.comprod` - do we want a `/` in between the two place holders?

Reply:
I think the test is trying to capture that we using `URL.hostname` only extracts up to `<first part of url>.com` based on the rest of the test - `Utils.AWS_RUM_ENDPOINT`'s hostname is `rumservicelambda.us-west-2.amazonaws.com`, and we are checking even if it's `.comprod`, the `signedRequest`'s hostname does not include `prod`

Reply:
`const test = new URL(
            `${endpoint.toString()}${'prod'}`)
console.log('test ', test)` leads to `"test ",  URL: "https://rumservicelambda.us-west-2.amazonaws.com/prod" `

Reply:
related to https://github.com/aws-observability/aws-rum-web/pull/156#discussion_r874838260